src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -274,3 +286,41 @@ def get_storage_blob_uri(cli_ctx, storage):
             raise CLIError('{} does\'t exist.'.format(storage))
         storage_uri = storage_account.primary_endpoints.blob
     return storage_uri
+
+
+def get_sku_dict_from_string(sku_string):
```

Comment:
i would suggest you do like `update_write_accelerator_settings` does to stay consistent on processing those values.